askama_parser/src/node.rs

```diff
@@ -520,7 +520,13 @@ impl<'a> Macro<'a> {
                 |i| s.tag_block_end(i),
             ))),
         ));
-        let (i, (pws1, _, (name, params, nws1, _))) = start(i)?;
+        let (j, (pws1, _, (name, params, nws1, _))) = start(i)?;
+        if name == "super" {
```

Comment:
I guess all [reserved keywords](https://doc.rust-lang.org/reference/keywords.html) should be restricted? What do you think?

Reply:
I think that'd be better indeed.